### PR TITLE
add table accessor.

### PR DESCRIPTION
**Pull request by vwxyzh**

### CODE REVIEW

File: src/Tests/StorageTest/TableTest.cs [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                                Code = i,
+                                Text = dt.ToString("yyyyMMddHHmmss"),
+                            }).ToList();
+
+            await table.BatchInsertAsync(entities);
+            Assert.Collection(
+                entities,
+                Enumerable.Repeat((Action<TestEntity>)(entity => Assert.NotNull(entity.ETag)), Count).ToArray());
+
+            var retrived = await table.QueryAsync(
```
Comment (wanlwanl) [inline src/Tests/StorageTest/TableTest.cs:75]: typo

File: src/Libs/Storage/Tables/TableQueryExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+using System.Linq.Expressions;
+using System.Reflection;
+
+using Microsoft.Azure.Cosmos.Table;
+
+namespace Azure.SignalRBench.Storage
+{
+    public static class TableQueryExtensions
+    {
+        public static TableQuery<T> Where<T>(this TableQuery<T> query, Expression<Func<T, bool>> predicate)
```
Comment (wanlwanl) [inline src/Libs/Storage/Tables/TableQueryExtensions.cs:14]: unused?

Comment (vwxyzh) [reply src/Libs/Storage/Tables/TableQueryExtensions.cs:14]: test by Linq query.

Comment (wanlwanl) [reply src/Libs/Storage/Tables/TableQueryExtensions.cs:14]: Got it

**Pull request merged**